src/Cache/Cache.php

```diff
@@ -168,7 +168,18 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
```

Comment:
can we make all engines have a fallback? Just set the fallback to the `Null` engine, this way we avoid the check